ui/main/src/app/api/handlebars.api.ts

```
@@ -0,0 +1,32 @@
+
+import {LogOption, LoggerService as logger} from '../business/services/logs/logger.service';
+
+declare const opfab: any;
+
+export class HandlebarsAPI {
+    public static async getHandlebarHelpers() {
+        return undefined;
+    }
+
+    public static init() {
+        const self = this;
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)